Readability.js

```diff
@@ -462,6 +477,37 @@ Readability.prototype = {
     return node && node.nextElementSibling;
   },
 
+  /**
+   * Like _getNextNode, but for DOM implementations with no
+   * firstElementChild/nextElementSibling functionality...
```

Comment:
Does this only concern jsdom? If so we might want to revisit using it at all if it's broken…